Assets/HoloToolkit-Tests/Input/Scenes/SpeechInputSource.unity

```diff
@@ -13,7 +13,7 @@ OcclusionCullingSettings:
 --- !u!104 &2
```

Comment:
In the new system, you specify all your keywords on the `SpeechInputSource` and then assign them on your `SpeechInputHandler`.